app/views/notification_mailer/notification_email.html.slim

```diff
@@ -0,0 +1,13 @@
+h4
+  = I18n.t('notification_mailer.notification.greeting') + ', ' +  @user.name
+
+= sanitize(@options[:content])
+
+- if @options[:html_link]
```

Comment:
Instead of this, you might want to consider using the notification email as a template, and each source of notifications will provide a template for the body.